s3/replication/manager/src/s3replicationmanager/__main__.py

```diff
@@ -29,7 +29,9 @@ def setup_args(parser):
         '--configfile',
         type=str,
         metavar='path',
-        help='Path to configuration file(format: yaml)')
+        help='Path to configuration file(format: yaml).\n'
```

Comment:
how about this?
```
help="""Path to configuration file(format: yaml).
e.g: ~/.cortxs3/credentials.yaml """
```

Reply:
Docstring containing both lines is not represented properly in output. so I am using, single quotes for each line.

Output would be as below :
![image](https://user-images.githubusercontent.com/49739659/125902746-8e04d254-6725-476e-aa88-b9e56e710266.png)